pyfar/plot/_utils.py

```diff
@@ -277,3 +277,22 @@ def _deal_time_units(unit='s'):
         factor = 1
         string = ''
     return factor, string
+
+
+def _log_prefix(fft_norm):
```

Comment:
Would it make sense to also move this code inside this function
```python
if log_prefix is None:
    if isinstance(signal, Signal):
        log_prefix = _utils._log_prefix(signal.fft_norm)
    else:
        log_prefix = 20
```
Otherwise we would have to repeat it in every freq-like plot function

Reply:
I changed the function to be called with the signal and kept the None check outside. It could be included in a general dB calculation function?